assets/js/modules/adsense/datastore/report.test.js

```diff
@@ -56,7 +53,8 @@ describe( 'modules/adsense report', () => {
 
 	afterEach( () => {
 		unsubscribeFromAll( registry );
-		apiFetchSpy.mockRestore();
+		fetchMock.restore();
+		fetchMock.mockClear();
```

Comment:
In addition to other comments earlier about these, this should be something we can add to all tests globally in the setup rather than adding into each.

Reply:
Yeah, we can do this in `setup-before-after.js`